scripts/test.sh

```diff
@@ -11,7 +11,9 @@ for crate in $crates; do
     echo "Testing crate: $crate..."
 
     # Always test the crate itself first, without any features added.
-    $CARGO ${CARGO_COMMAND} -p $crate
+    if [[ "$crate" != "utoipa-gen" ]]; then
+        $CARGO ${CARGO_COMMAND} -p $crate
```

Comment:
Do you mind if you wrap the variable with double quotes. I have a shellcheck enabled and it does complain and rightfully so that unquoted variables might get globbed or splitted. Like follows but all entries that are not double quoted
```suggestion
        $CARGO "${CARGO_COMMAND}" -p $crate
```

Reply:
Though I can also wrap then later as well

Reply:
Without quotes, the following works
```
CARGO_COMMAND="test --offline" ./scripts/test.sh
```
I see the shellcheck warning and will investigate.

I wonder why it doesnt also complain about `$CARGO`.

Reply:
> I wonder why it doesnt also complain about `$CARGO`.

That's interesting. My version does not error about that. I am actually using the bash lsp server.